interface/backend/images/models.py

```diff
@@ -23,3 +47,4 @@ class ImageLocation(models.Model):
     y = models.PositiveSmallIntegerField(help_text='Voxel index for Y axis, zero-index, from top left')
 
     z = models.PositiveSmallIntegerField(help_text='Slice index for Z axis, zero-index')
+
```

Comment:
Please be careful of checking your commits before pushing... You've read about Van Halen's brown M&M's ider, right? :)